cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -399,18 +399,18 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Builds Ballerina module(s)/file and produces an executable jar file(s). \n");
+        out.append("build a Ballerina module(s)/file and produce an executable .jar file(s). \n");
```

Comment:
This is the same as what is in the help files for description right?

I think we should have both be the same, and starting with an upper case letter seems better. 

I also think when it comes to description, the current form (Build**s**) is OK.

Reply:
Changed as follows in here and in the `ballerina-build.help` file:

"Build Ballerina modules/file and generate executable .jar file(s)."